_rules/text-contrast-afw4f7.md

```diff
@@ -394,9 +388,21 @@ This text is part of a label of a [disabled][] widget, because it is in a `label
 </div>
 ```
 
-[disabled]: #disabled-element 'Definition of disabled'
-[visible]: #visible 'Definition of visible'
-[text node]: https://dom.spec.whatwg.org/#text 'Definition of text node'
-[purely decorative]: https://www.w3.org/TR/WCAG21/#dfn-pure-decoration 'Definition of purely decorative'
-[presentational roles conflict resolution]: https://www.w3.org/TR/wai-aria-1.1/#conflict_resolution_presentation_none 'Presentational Roles Conflict Resolution'
-[semantic role]: #semantic-role 'Definition of semantic role'
+[accessible name]: #accessible-name 'Definition of Accessible Name'
+[background colors]: #background-colors-of-text 'Definition of Background color of text'
+[disabled]: #disabled-element 'Definition of Disabled'
+[foreground colors]: #foreground-colors-of-text 'Definition of Foreground color of text'
+[highest possible contrast]: #highest-possible-contrast 'Definition of Highest possible contrast'
+[larger scale text]: #large-scale-text 'Definition of Large scale text'
+[semantic role]: #semantic-role 'Definition of Semantic role'
+[visible]: #visible 'Definition of Visible'
+[ancestor]: https://dom.spec.whatwg.org/#concept-shadow-including-ancestor 'DOM, ancestor, 2020/07/23'
+[child]: https://dom.spec.whatwg.org/#concept-tree-child 'DOM, child, 2020/07/23'
+[text node]: https://dom.spec.whatwg.org/#text 'DOM, text node, 2020/07/23'
+[origins]: https://www.w3.org/TR/css3-cascade/#cascading-origins 'CSS 3, origin'
+[user origin]: https://www.w3.org/TR/css3-cascade/#cascade-origin-user 'CSS 3, user origin'
+[flat tree]: https://drafts.csswg.org/css-scoping/#flat-tree 'CSS draft, flat tree, 2020/07/23'
+[presentational roles conflict resolution]: https://www.w3.org/TR/wai-aria-1.1/#conflict_resolution_presentation_none 'WAI-ARIA, Presentational Roles Conflict Resolution'
+[purely decorative]: https://www.w3.org/TR/WCAG21/#dfn-pure-decoration 'WCAG 2.1, Purely decorative'
+[human language]: https://www.w3.org/TR/WCAG21/#dfn-human-language-s 'WCAG 2.1, Human language'
+[sc143]: https://www.w3.org/TR/WCAG21/#contrast-minimum 'WCAG 2.1, Success criterion 1.4.3 Contrast (Minimum)'
```

Comment:
```suggestion
[accessible name]: #accessible-name 'Definition of Accessible Name'
[ancestor]: https://dom.spec.whatwg.org/#concept-shadow-including-ancestor 'DOM, ancestor, 2020/07/23'
[background colors]: #background-colors-of-text 'Definition of Background color of text'
[child]: https://dom.spec.whatwg.org/#concept-tree-child 'DOM, child, 2020/07/23'
[disabled]: #disabled-element 'Definition of Disabled'
[flat tree]: https://drafts.csswg.org/css-scoping/#flat-tree 'CSS draft, flat tree, 2020/07/23'
[foreground colors]: #foreground-colors-of-text 'Definition of Foreground color of text'
[highest possible contrast]: #highest-possible-contrast 'Definition of Highest possible contrast'
[human language]: https://www.w3.org/TR/WCAG21/#dfn-human-language-s 'WCAG 2.1, Human language'
[larger scale text]: #large-scale-text 'Definition of Large scale text'
[origins]: https://www.w3.org/TR/css3-cascade/#cascading-origins 'CSS 3, origin'
[presentational roles conflict resolution]: https://www.w3.org/TR/wai-aria-1.1/#conflict_resolution_presentation_none 'WAI-ARIA, Presentational Roles Conflict Resolution'
[purely decorative]: https://www.w3.org/TR/WCAG21/#dfn-pure-decoration 'WCAG 2.1, Purely decorative'
[text node]: https://dom.spec.whatwg.org/#text 'DOM, text node, 2020/07/23'
[sc143]: https://www.w3.org/TR/WCAG21/#contrast-minimum 'WCAG 2.1, Success criterion 1.4.3 Contrast (Minimum)'
[semantic role]: #semantic-role 'Definition of Semantic role'
[user origin]: https://www.w3.org/TR/css3-cascade/#cascade-origin-user 'CSS 3, user origin'
[visible]: #visible 'Definition of Visible'
```
I'm in favour of keeping definitions list order alphabetically as it makes it easier for further edit to figure out if something is here or not.
We might need a test for that if we really want to enforce it, however…